invenio_cli/commands/requirements.py

```diff
@@ -150,6 +150,20 @@ def check_imagemagick_version(cls, major, minor=-1, patch=-1, exact=False):
                 status_code=1,
             )
 
+    @classmethod
+    def check_git_version(cls, major, minor=-1, patch=-1, exact=False):
+        """Check the ImageMagick version."""
```

Comment:
docstring does not match the method purpose

Reply:
Ups, fixed it